sopel/modules/reload.py

```diff
@@ -11,20 +11,19 @@
 import collections
 import sys
 import time
-from sopel.tools import iteritems
+from sopel.tools import iteritems, get_raising_file_and_line
 import sopel.loader
 import sopel.module
 import subprocess
+import os
 
 
-@sopel.module.nickname_commands("reload")
```

Comment:
I'm not a fan of changing the command behavior just for the hell of it. `Sopel: reload modulename` should continue to work.